rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRest.java

```diff
@@ -117,6 +120,10 @@
 
     private static final Pattern PATTERN = Pattern.compile("^[^:]*:(.*)");
 
+    protected static final String NODE_CONFIG_TAGS_KEY = "nodeTags";
```

Comment:
if it's a JVM property, I prefer "node.tags"

Reply:
Actually, I don't understand how it is used

Reply:
Ok, now I understand, it's a key in the json configuration for the nodes. so nodeTags is ok